main.go

```diff
@@ -156,7 +156,7 @@ func main() {
 	}
 
 	// Initialize the shared authentication manager for service containers to authentication to the agent.
-	authm := resource.NewAuthenticationManager(cfg.GetFileSyncServiceAuthPath())
+	authm := resource.NewAuthenticationManager(cfg.GetFileSyncServiceAuthPath(), "/var/horizon/secrets")
```

Comment:
This needs to be configurable and overrideable by an env var.

Reply:
Fixed